docker-compose-production.yml

```diff
@@ -0,0 +1,40 @@
+app:
+  build: .
+web:
+  extends:
+    service: app
+  command: gunicorn -b 0.0.0.0:8000 --reload measurements.wsgi --workers 4 --timeout 120
+  ports:
+    - "8000"
+  links:
+    - redis
+    - rabbitmq
+https-portal:
+  image: steveltn/https-portal:1.0.0
+  ports:
+    - "80:80"
+    - "443:443"
+  links:
+    - web
```

Comment:
As I said, if you don't find a way to do that, just rip the db: container definition out of the base docker-compose file and place it inside of dev, testing, staging and do not put it in production.

In production put the https-portal definition and voila, done, no need for a docker-compose-production.

Pasting what I wrote also on IRC for futher context.
```
09:28 < hellais> docker-compose.yml is the base docker compose file (this is shared by EVERY deployment target)
09:29 < hellais> config/$TARGET.yml is the overriden docker compose file that contains changes specific to that target. You can put in here ANYTHING,
                 including the creation of extra containers, linking to other containers, setting up port mapping, etc.
09:30 < hellais> config/$TARGET.env is the environment file definition for a specific target. This contains ONLY environment variable definitions
09:30 < hellais> config/base.env contains the environment variables shared by every target.
```